packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/EnrichDatasetResponseHandler.java

```diff
@@ -125,10 +135,6 @@ public TaskResponse onSuccess(final TaskResponse resp) {
 				metadata.put("description", description);
 				metadata.put("unit", unit);
 
-				// Based on the name, description, fetch the best grounding available and add it to the metadata
-				// final DKG grounding = dkgService.knnSearchEpiDKG(0, 1, 1, name + " " + description, null)
-				// metadata.put("grounding", grounding);
-
```

Comment:
Why removing this comment.

Reply:
because its a comment... and it's broken anyway.